docs/UserGuide.md

```diff
@@ -109,66 +158,102 @@ Examples:
 
 Adds a class to an existing student's record.
 
-Format: `addc STUDENT_ID CLASS_NAME`
+Format: `addc i/IC_NUMBER s/SUBJECT`
 
-* `STUDENT_ID` should be a 5-digit number.
-* `CLASS_NAME` should be a combination of Subject and Number (e.g., Science1).
+* `IC_NUMBER` is compulsory and should follow the format of Singaporean IC and FIN numbers (e.g., S1234567A).
+* `SUBJECT` is compulsory, and can add multiple subjects (e.g. Science).
 
 Examples:
-* `addc 12345 Science1`
+* `addc i/S1234567A s/Science`
+
+![addc Success Message](/images/addc.png)
 
 ### Sorting students : `sort`
 
 Sorts the list of students based on a specified field.
 
-Format: `sort FIELD`
+Format: `sort s/FIELD`
 
 * `FIELD` can be either `name` or `class`.
 
 Examples:
-* `sort name`
-* `sort class`
+* `sort s/name`
+* `sort s/class`
+
+![Sort Success Message](/images/sort.png)
 
 ### Clearing all entries : `clear`
 
 Clears all student entries from the system.
 
 Format: `clear`
 
+![Clear Success Message](/images/clear.png)
+
 ### Viewing help : `help`
 
 Shows a message explaining how to access the help page.
 
 Format: `help`
 
+![Help Success Message](/images/help.png)
+
 ### Exiting the program : `exit`
 
 Exits the program.
 
 Format: `exit`
 
-### Saving the data
+---
 
-AcademyAssist data are saved in the hard disk automatically after any command that changes the data. There is no need
-to save manually.
+## Data Management in AcademyAssist
 
-### Editing the data file
+Welcome to the Data Management section of AcademyAssist! Here, we'll explain how your important information
+is stored and managed.
 
-AcademyAssist data are saved automatically as a JSON file `[JAR file location]/data/academyassist.json`. Advanced users
-are welcome to update data directly by editing that data file.
+### How Your Data is Saved
+
+Don't worry about constantly saving your work - AcademyAssist has got you covered! Here's what you need to know:
+
+1. **Automatic Saving**: Every time you make a change, like adding a new student or updating information,
+AcademyAssist saves it right away. You don't have to do anything!
+
+2. **Where Your Data Lives**: All your information is safely stored on your computer in a special file.
+It's like a digital filing cabinet just for AcademyAssist.
+
+3. **Easy to Understand**: We use a format called JSON to store your data. While you don't need
+to know the details, it's designed to be easy for both computers and humans to read if needed.
+
+### Keeping Your Data Safe
+
+Your information is important, so here are some tips to keep it secure:
+
+1. **Make Backups**: It's always a good idea to have an extra copy of your data. Once a week, 
+why not copy your AcademyAssist file to a USB drive or cloud storage?
+
+2. **Be Careful with Editing**: While it's possible to open and edit the data file directly, we don't recommend 
+it unless you're very comfortable with computers. It's safer to make changes through the AcademyAssist program.
```

Comment:
"unless you're very comfortable with computers" may be a bit ambiguous, perhaps can change to unless you are an expert in CLI applications